merkletree/tb.go

```diff
@@ -25,3 +29,15 @@ func (tb *TemporaryBinding) Serialize(strSig []byte) []byte {
 	tbBytes = append(tbBytes, tb.Value...)
 	return tbBytes
 }
+
+// Verify verifies the issued temporary binding whether
+// it was inserted as promised or not. This method should be
+// called right after the directory has been updated, i.e., at the next epoch.
+func (tb *TemporaryBinding) Verify(ap *AuthenticationPath) bool {
```

Comment:
We already have a discussion over this in https://github.com/coniks-sys/coniks-go/pull/100. Basically, that `epoch` field is implied by the STR signature included in the TB.